crates/libcontainer/src/container/builder.rs

```diff
@@ -37,6 +37,9 @@ pub struct ContainerBuilder<'a> {
 /// .build();
 /// ```
 impl<'a> ContainerBuilder<'a> {
+    /// ERR_INVALID_ID is container id validate error
+    const ERR_INVALID_ID: &str = "invalid container ID format";
```

Comment:
Can you please use the `anyhow` error and result, following how the rest of the code is doing? This is actually a good example where we should use `thiserror`, but we have not done this yet.

Reply:
Already using `anyhow` error and Result, I can submit pr later to use `thiserror`.